pkgs/development/interpreters/erlang/generic-builder.nix

```
@@ -85,12 +98,13 @@ in stdenv.mkDerivation ({
     ++ optional enableThreads "--enable-threads"
     ++ optional enableSmpSupport "--enable-smp-support"
     ++ optional enableKernelPoll "--enable-kernel-poll"
-    ++ optional enableHipe "--enable-hipe"
+    ++ optional hipeSupport "--enable-hipe"
```

```suggestion
    ++ optional (enableHipe && !isCross) "--enable-hipe"
```

But honestly I would prefer an asset and throw here because people will be confused why hipe is disabled when cross compiling.